sdk/sdk-core/aar/src/main/java/mil/emp3/core/storage/StorageManager.java

```diff
@@ -1072,15 +1071,15 @@ private void executeTransaction(TransactionList transactionList, Object userCont
      * @param children
      * @param parentWrapper
      */
-    private void getChildren(java.util.Set<IContainer> children,
+    private void getChildren(Set<IContainer> children,
                                        StorageObjectWrapper parentWrapper) {
```

Comment:
could you line up the StorageObjectWrapper with the Set<IContainer> parameter? The formatting is wonky due to the importing of the package change

Reply:
No, you are wasting my time

Reply:
I always harp on people to keep their syntax consistent, but not necessarily lined-up.  If you are comfortable with this particular example that is fine, but I wouldn't call readability a waste of time in all cases.